### PR TITLE
Setup Github action

### PR DESCRIPTION
**Pull request by mirland**

### CODE REVIEW

File: .github/workflows/compile.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run detekt
+      run: ./gradlew detekt
```
Comment (f7deleon) [inline .github/workflows/compile.yml:13]:
```suggestion
      run: ./gradlew detektCheck
```

**Review by f7deleon — APPROVED** [review]

File: .github/workflows/compile.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run detekt
+      run: ./gradlew detektCheck
+    - name: Lints
+      run: /gradlew lintDebug
```
Comment (f7deleon) [inline .github/workflows/compile.yml]:
```suggestion
      run: ./gradlew lintDebug
```

File: .github/workflows/compile.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    - name: Compile
+      run: /gradlew assembleDebug
```
Comment (f7deleon) [inline .github/workflows/compile.yml]:
```suggestion
      run: ./gradlew assembleDebug
```

**Comment by mirland** [comment]
It was not working because it require merge the previous pull

File: .github/workflows/compile.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run detekt
+      run: ./gradlew detektCheck
```
Comment (mirland) [inline .github/workflows/compile.yml]: I think it's wrong

File: .github/workflows/compile.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    - name: Lints
+      run: ./gradlew lint
```
Comment (mirland) [inline .github/workflows/compile.yml]: Choose debug to be faster

File: .github/workflows/compile.yml (hunk 4 above)
Comment (f7deleon) [reply .github/workflows/compile.yml]: will revert the commit so it goes back to the first config, didn't think that the other pull was needed

Comment (mirland) [reply .github/workflows/compile.yml]: Yes, it was my bad. I had to tell you that in the description but I forgot it!

**Pull request merged**